packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -69,18 +70,20 @@ const MotionWelcome = styled(motion.div)`
 `;
 
 const LinksContainer = styled.div`
-    position: absolute;
     bottom: 0;
     display: flex;
-    margin: 24px 0;
+    margin: ${spacingsPx.xl};
+    align-items: center;
+    flex-flow: row wrap;
+    gap: 14px;
```

Comment:
Maybe could we use `spacingsPx.md` here? Does it look good?

Reply:
Very little visual change of the gap, IMHO both options are ok, see 14px vs 16 px (difference almost unseen):
![gap](https://github.com/user-attachments/assets/8592114b-9744-484b-9258-930fb59b319e)
I think we can safely use `spacingsPx.md` here.